picard/tagger.py

```diff
@@ -334,7 +334,7 @@ def load_to_picard(self, items):
             if not parsed.scheme:
                 files.append(item)
             elif parsed.scheme == "file":
-                files.append(item.replace("file://", ''))
+                files.append(parsed.path)
```

Comment:
Actually a bad idea, sorry.
It will fail with paths like "file:///some#thing"

Reply:
Okay, better idea: 

`item[7:]`